source/python/README.md

```diff
@@ -1638,7 +1641,7 @@ This will be less than or equal to zero.
 **Calculates the inverse of a rotation matrix.**
 
 Given a rotation matrix that performs some coordinate transform,
-this function returns the matrix that reverses that transform.
+this function returns the matrix that reverses that trasnform.
```

Comment:
I fixed this mistake, trasnform -> transform, recently. I don't know how it came back in your changes. That is strange.

Reply:
I really don't why it touches python documentations!